openid4vc-high-assurance-interoperability-profile-sd-jwt-vc-1_0.md

```diff
@@ -215,6 +223,57 @@ This is an example of a Wallet Instance Attestation:
     * In the `input-descriptors` object, `id`, `name`, `purpose`, `group`, `format`, and `constraints` properties MUST be supported. In the `constraints` object, `limit_disclosure`, and `fields` properties MUST be supported. In the `fields` object, `path` and `filter` properties MUST be supported. A `path` MUST contain exactly one entry with a static path to a certain claim. A `filter` MUST only contain `type` elements of value `string` and `const` elements.
     * In the `submission_requirements` object, `name`, `rule (`pick` only)`, `count`, `from` properties MUST be supported.
 
+# OpenID for Verifiable Presentations over W3C Digital Credentials API
+
+* MUST support Annex A in [@!OIDF.OID4VP] that defines how to use OID4VP over the W3C Digital Credentials API.
+  * Annex A.3.1. in [@!OIDF.OID4VP] MUST NOT be used. OID4VP requests over the W3C Digital Credentials API MUST be signed.
```

Comment:
Why is this a requirement? Especially considering that it's up to the wallet how to continue when it cannot authenticate the RP.

Reply:
This is a requirement that comes from eIDAS 2.0 regulation, that mandates that the wallet must authenticate the RP/Verifier.

Reply:
ISO 18013-5 says: "An mDL may require mdoc reader authentication (see 9.1.4) before releasing data elements not marked as mandatory in Table 5. An mDL shall not require mdoc reader authentication as a precondition for returning any of the mandatory data elements. An mDL may offer functionality to the mDL holder to pre-authorize the return of mandatory data elements selected by the mDL holder to mDL readers using mdoc reader authentication."

and ISO 18013-7 says "The mDL data model descriptions and requirements in ISO/IEC 18013-5 shall apply in this document with the following exception: an mDL may require mdoc reader authentication as a precondition for the release of any of the mandatory data elements."

so in 18013-7, technically, mdoc reader authentication (eg verifier authentication) can be mandated as a pre-condition to release both mandatory and not mandatory data elements.

Reply:
If HAIP decides not mandate signing the request, that would not meet EC's requirements, we would need to alert EC as soon as possible, so that they can plan accordingly.

Reply:
I support mandating the signing of the request

Reply:
discussed in the WG. seems to be support to allow both signed and unsigned request, to make this profile global. 

Separate questions are:
-  if we want to make any of these mandatory to support for the wallet?
- for EU, separate profiling will be required, if they want to make signing mandatory.
- Might want to add additional sentence of the value of the signing of the request

will open a separate issue on this.

```suggestion
```

Reply:
#129